core/src/main/java/bio/terra/pearl/core/service/export/formatters/module/EnrolleeRelationFormatter.java

```diff
@@ -0,0 +1,44 @@
+package bio.terra.pearl.core.service.export.formatters.module;
+
+import bio.terra.pearl.core.model.participant.EnrolleeRelation;
+import bio.terra.pearl.core.service.export.EnrolleeExportData;
+import bio.terra.pearl.core.service.export.formatters.item.PropertyItemFormatter;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class EnrolleeRelationFormatter extends ModuleFormatter<EnrolleeRelation, PropertyItemFormatter<EnrolleeRelation>> {
+    private static final String MODULE_NAME = "proxy";
+    private static final List<String> INCLUDED_PROPERTIES =
+        List.of("enrollee.shortcode", "relationshipType", "beginDate", "endDate");
+
+    public EnrolleeRelationFormatter() {
+        itemFormatters = INCLUDED_PROPERTIES.stream()
+                .map(propName -> new PropertyItemFormatter<>(propName, EnrolleeRelation.class))
+                .collect(Collectors.toList());
+        // we have to handle kitType separately because we'll need to match it to the kitType name
+        moduleName = MODULE_NAME;
+        displayName = "Proxy Relations";
+    }
+
+    @Override
+    public Map<String, String> toStringMap(EnrolleeExportData enrolleeData) {
```

Comment:
let's add a comment at the top of this method that this is a near-duplicate of toStringMap in KitRequestFormatter so that if we end up with another formatter like this, we can extract helper methods